### PR TITLE
[IMP] Add SSL support for Flask

### PR DESCRIPTION
**Pull request by rousseldenis**
Superseeds #27 

Depends on:

* [x] #29

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @rousseldenis. First, thanks for all this recent PR on pywebdriver. Happy to see Acsone working on that project. 
2 remarks : 
- a general one, I don't understand why all the code is duplicated. (in pywebdriverd and in pywebdriver.py). Is it possible to make a unique function to call ?
- could you add in the readme part, a section to mention how to set ssl ? 

otherwise, LGTM. Thanks.

**Comment by rousseldenis** [comment]
@legalsylvain Yes, we are using this for nearly a year now and I'm also personnaly interested in, so I wanted to move forward on the work done by a former colleague.



> * a general one, I don't understand why all the code is duplicated. (in pywebdriverd and in pywebdriver.py). Is it possible to make a unique function to call ?

I don't know why it was done like that. I will explore and let you informed.



> * could you add in the readme part, a section to mention how to set ssl ?

Of course, I will.

**Comment by rousseldenis** [comment]
> * a general one, I don't understand why all the code is duplicated. (in pywebdriverd and in pywebdriver.py). Is it possible to make a unique function to call ?

I think this has been done to work with https://github.com/akretion/pywebdriver/pull/26 as it should find the python file (pywebdriverd.py) and not the agnostic one. Maybe also to keep compatibility with other interactions.

I will try to do it with a simple import instead.

**Comment by rousseldenis** [comment]
@legalsylvain README updated

**Comment by rousseldenis** [comment]
@legalsylvain @PierrickBrun This one is rebased and ready

File: pywebdriverd [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+import os
```
Comment (PierrickBrun) [inline pywebdriverd]:
```suggestion
import os
import sys
```

File: pywebdriverd.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```
Comment (PierrickBrun) [inline pywebdriverd.py]: I think we need to remove this

**Comment by PierrickBrun** [comment]
Already included

**Pull request closed without merging**